rdmo/projects/validators.py

```diff
@@ -7,18 +9,42 @@
 from rdmo.core.utils import human2bytes
 
 
-class ValueValidator:
+class ValueConflictValidator:
 
     requires_context = True
 
     def __call__(self, data, serializer):
-        if data.get('value_type') == VALUE_TYPE_FILE:
+        if serializer.instance:
+            # for an update, check if the value was updated in the meantime
+            updated = serializer.context['view'].request.data.get('updated')
+            if updated is not None and parse_datetime(updated) < serializer.instance.updated:
```

Comment:
There is still a situation where this could create problems with regular use of RDMO by only one user. When the client posts the same value again, before the updated value "returned" to the client (due to a slow network) the validator would raise an error. We need to talk about this, thats why I am leaving this note here.